configsnap

```diff
@@ -712,12 +712,7 @@ if os.path.exists('/usr/bin/php'):
     is_php_detected = True
     report_info("Copying PHP related files...")
     run.copy_file('/etc/php.ini', fail_ok=True)
-    destdir = os.path.join(workdir, "php.d/")
-    if not os.path.exists(destdir):
-        os.makedirs(destdir)
-
-    for filename in glob.glob('/etc/php.d/*'):
-        run.copy_file(filename, destdir, fail_ok=False)
+    run.copy_dir('/etc/php.d/', fail_ok=False, sort=False)
```

Comment:
This fails on Debian Stretch at least, with php-fpm installed and using `/etc/php/...` with no `/etc/php.d` present.
That created a false positive when configsnap runs.
> Copying PHP related files...
 Directory /root/configsnap/test2/configsnap/php.d does not exist

Works OK if /etc/php.d present. Not directly impacted by the PR which is just a rewrite of the same paradigm. Will create issue for this.